src/java/lesson4/NadyaBryashchey_Task1.java

```diff
@@ -0,0 +1,76 @@
+package java.lesson4;
+import java.util.Random;
+import java.util.Scanner;
+
+/* Task 1
+Создать последовательность случайных чисел, найти и вывести наибольшее из них.
+(случайные числа можно получить из класса Random)
+ */
+
+public class NadyaBryashchey_Task1 {
+    public static int[] RANDOM_ARRAY;
+    public static Random globalRandom = new Random();//works everywhere
+    public static int sizeOfArray;
+    public static Scanner scanner = new Scanner(System.in);
+
+
+    public static void main(String[] args) {
+        //получить розмер масива
```

Comment:
Почему не с большой буквы предложение
```suggestion
        //  Получить розмер масива
```

Reply:
сделал , проверил вордом синтаксис